packages/web-client/app/routes/application.ts

```diff
@@ -2,8 +2,9 @@ import Route from '@ember/routing/route';
 import RouterService from '@ember/routing/router-service';
 import { LocationService } from '../services/location';
 import { inject as service } from '@ember/service';
-import '@cardstack/web-client/css/variables.css';
 import config from '@cardstack/web-client/config/environment';
+import '@cardstack/web-client/css/variables.css';
+import '@cardstack/web-client/css/card-space.css';
```

Comment:
I wonder if there's a good way to dynamically load CSS when we dynamically render a route template, so we don't have to load this on the application route.